cairo/src/instructions/block_information.cairo

```diff
@@ -126,10 +129,24 @@ namespace BlockInformation {
         Stack.push_uint128(0);
         jmp end;
 
-        blobbasefee:
+        blob_base_fee:
         let stack = cast([fp - 6], model.Stack*);
-        Stack.push_uint128(0);
-        jmp end;
+        let evm = cast([fp - 3], model.EVM*);
+        let range_check_ptr = [fp - 9];
+        with_attr error_message("Excess blob gas is not set") {
+            assert evm.message.env.excess_blob_gas.is_some = 1;
+        }
```

Comment:
if we're in 'cancun' mode, there should always be an excess blob gas.

this is because the block type is modeled after reth, and not EELS: see issue https://github.com/kkrt-labs/keth/issues/246